packages/backend/src/graphql/queries/get-executions.ts

```diff
@@ -10,13 +10,12 @@ const getExecutions: QueryResolvers['getExecutions'] = async (
   context,
 ) => {
   const filterBuilder = (builder: ExtendedQueryBuilder<Execution>) => {
-    builder.where('test_run', 'FALSE')
+    builder.where('test_run', false)
     builder.where('flow_id', params.flowId)
 
-    if (!('status' in params)) {
+    if (params.status === null) {
       builder.whereNull('status')
-    }
```

Comment:
when status filter is set to waiting, the status passed to backend is actually undefined. so comparing param.status === null will break the `waiting` filter